frontend/src/app.css

```diff
@@ -21,9 +21,12 @@ body {
 }
 
 .content {
```

Comment:
please avoid using global css wherever possible. If these styles are required somewhere, please put it in their respective css module else remove it

Reply:
ok i'll try!

Reply:
I can't seem to override the global attribute. I tried:
* .crd .content
* .crd > .content
* using !important in the styles

Is there anyway to tackle this?